src/app/components/common/filters/filters.component.ts

```diff
@@ -102,4 +102,14 @@ export class FiltersComponent {
     }
     return filter.values.filter((value) => value.toLowerCase().includes(searchTerm.toLowerCase()));
   }
+
+  isChecked(name: string, value: string) {
+    if (this.selectedFilters && this.selectedFilters.length > 0) {
+      const index = this.selectedFilters.findIndex((item) => item.name === name);
+      if (this.selectedFilters[index]) {
+        return this.selectedFilters[index].values.includes(value) ? true : false;
+      }
+    }
+    return false;
```

Comment:
Not sure if this could be as simple as this?

```suggestion
    if (this.selectedFilters.length) {
      return Boolean(this.selectedFilters.find((item) => item.name === name)?.values?.includes(value));
    }
    return false;
```